pkg/asset/tls.go

```diff
@@ -82,6 +82,7 @@ func newAPIKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey, altNa
 		"kubernetes.default",
 		"kubernetes.default.svc",
 		"kubernetes.default.svc.cluster.local",
+		"localhost",
```

Comment:
Will the admission controller only use localhost? I'd prefer not to have this as a SAN if possible